contacts-app/src/components/ColumnHeader.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+type Props = {
+    children: any
```

Comment:
Note 2

Reply:
When using typescript, avoid as much as possible to use any. or just don't use typescript at all
children here should be typed ReactNode